app/templates/power-ups/client-library.html

```diff
@@ -86,45 +50,116 @@
+  Read more about data storage in the <a ui-sref="power-ups.topics({'#':'data-storage'})">topics section</a>.
+  </li>
 	<li><strong>get(scope, visibility, name, defaultValue)</strong> Used to get all of the stored data for the Power-Up</li>
 	<li><strong>attach({url:URL, name:Name})</strong> Attach a new URL to the card in the current context.</li>
-	<li><strong>signUrl(URL)</strong> Sign a URL for use with <strong>attachment-sections</strong> only.</li>
-	<li><strong>localizeKey(key, data)</strong> Localize a string by key</li>
-	<li><strong>localizeKeys(keys)</strong> Localize an list of strings or objects</li>
-	<li><strong>board([requestedField1],[requestedField2],[…])</strong> Returns a promise with information about the board for the current context
+  <li><strong>signUrl(URL)</strong> Sign a URL for use with <strong>attachment-sections</strong> only.</li>
+  <li><strong>localizeKey(key, data)</strong> Localize a string by key</li>
+  <li><strong>localizeKeys(keys)</strong> Localize a list of strings or objects. You can read more about internationalizationa and localization in the <a ui-sref="power-ups.topics({'#':'internationalization'})">topics section</a>.</li>
+  <li><strong>board(field1, field2, ...)</strong>
+	Returns a promise with information about the board for the current context.
 	Valid fields include: 'id', 'name', 'url', 'shortLink'</li>
-	<li><strong>list([requestedField1],[requestedField2],[…])</strong> Returns a promise with information about the list for the current context
+	<li><strong>list(field1, field2, ...)</strong>
+	Returns a promise with information about the list for the current context
 	Valid fields include: 'id', 'name'</li>
-	<li><strong>card([requestedField1],[requestedField2],[…])</strong> Returns a promise with information about the card for the current context
-	Valid fields include: 'id', 'name', 'desc', 'due', 'cover', 'attachments', 'members', 'labels', 'url', 'shortLink'</li>
-	</ul>
-	<p>
-	<li><strong>t.closePopup</strong>, <strong>t.closeOverlay</strong>, <strong>t.iframe</strong> can only be called within an appropriate context.
-	</p>
-	<p>
+	<li><strong>card(field1, field2, ...)</strong>
+	Returns a promise with information about the card for the current context. The information available is only when a specific card is in context. For instance, this function will return information about a card successfully inside of the capabilities 'attachment-sections' and 'card-badges', but not 'show-settings' nor 'board-buttons'. The latter are not within the context of a single card.
+	Valid fields include: 'id', 'name', 'desc', 'due', 'closed', 'cover', 'attachments', 'members', 'labels', 'url', 'shortLink', 'idList'<br /><br />
+	Example usage:
+<pre>t.card('id', 'name', 'url').then(function(promiseResult){console.log(promiseResult)});
+// <- {id:"72hejh3iuruyr87rhiu", name: "Card Name", url: "https://trello.com/c/38UCRu1a/card-name"}</pre>
+	</li>
```

Comment:
A couple other fancy options here include:

`t.lists` - takes same args as `t.list` but gives back all open lists on the board
`t.member` - accepted fields: `'id', 'fullName', 'username'` - easy way to get information about active member
`t.cards` - takes same args as `t.card` but gives back all open cards on the board